test/test.spec.ts

```diff
@@ -94,6 +95,20 @@ describe('service', function () {
       // Assert
       expect(componentInstance).toEqual(componentInstance2);
     })
+
```

Comment:
Yea, I would just add a comment saying this should be refactored in the future.

Also, there are other categories of tests which test the communication at different layers which should also be added.  These are more strict forms of unit tests and it would be good to match the types of tests the the reports have.
For example there is section that only tests the messages to HPM, and would be named something like `dashboard.load sends POST /dashboard/load`.  Then there is other's that got to the promise level and say `dashboard.load returns promise that is resolved with 202 Accepted` or `...is rejected with 400 when given bad load configuration` etc.